src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Buldozer started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Buldozer ended its work");

```

Comment:
Why don’t you use `this.getClass().getSimpleName()` here? It's good practice to make your code consistent